standards-of-conduct.md

```diff
@@ -51,13 +51,22 @@ If anyone engages in unacceptable behavior, the conference organizers may take
 any action they deem appropriate including expulsion from the conference
 without warning or refund.
 
-## 6. What to do if you witness or are subject to unacceptable behavior
+## 6. Reporting Incidents
 
 If you are subject to unacceptable behavior, notice that someone else is being
-subject to unacceptable behavior, or have any other concerns, please notify
-[mailto:soc@perlconference.us](soc@perlconference.us) as soon as possible.
+subject to unacceptable behavior, or have any other concerns, please notify a
+member of the SoC Event Committee as soon as possible. If you do not know who
+these members are, ask any event organizer or volunteer for help. You may also
+email [mailto:soc@perlconference.us](soc@perlconference.us) to report an
+incident if you prefer.
 
-## 7. License and attribution
+## 7. Handling Incidents
```

Comment:
My suggestion is that this section doesn't really need to be in the SoC.   The existence of the document itself implies that it is intended to be used without having to spell that out.

Reply:
The main reason I think this should be here is so that when people are reading this on the website they see that we have this other document.

Reply:
In that case, how about this instead:

> 7.   Standards of Conduct Committee
> 
> Each event will establish a standards of conduct committee (SoC Committee) and a chairperson for that committee.    The point of contact for this committee will be published on the event website.    The SoC Committee will be responsible for receiving and handling all incident reports.   The SoC Committee is authorized to establish any supplemental procedures necessary to fulfill their role, provided that any such procedures are subject to approval by The Perl Foundation.

I believe this makes it clear that there may be extra procedures documented without necessarily spelling out what they may be.    This, in turn, will hopefully allow this document to exist from year to year without have to be constantly revised.

Reply:
I see what you're getting at. I just want to be able to link to this from the SoC on the website. So I think we can use your text and then just add a bit at the end of the SoC page saying "See our handling incidents page for blah blah blah".